ambari-server/src/main/java/org/apache/ambari/server/stack/StackDirectory.java

```diff
@@ -478,4 +526,52 @@ private void parseRoleCommandOrder() {
       LOG.error(String.format("Can not read role command order info %s", rcoFilePath), e);
     }
   }
+
+  /**
+   * Loads any classes found in the {@link #libraryDir} and creates a
+   * {@link URLClassLoader} for them.
+   */
+  private void parseLibraryClassLoader() {
+    if (null == libraryDir) {
+      return;
+    }
+
+    Path libraryPath = Paths.get(libraryDir);
+    if( Files.notExists(libraryPath) || !Files.isDirectory(libraryPath) ) {
```

Comment:
Does that work with a symlink pointing to a directory?

Reply:
No, it would not. However, we're not going to allow symlinks to the lib directory in the stack definition.